sdks/java/harness/build.gradle

```diff
@@ -55,21 +64,19 @@ description = "Apache Beam :: SDKs :: Java :: Harness"
 ext.summary = "This contains the SDK Fn Harness for Beam Java"
 
 dependencies {
-  dependOnShadedProjects.each {
-    implementation project(path: it, configuration: "shadow")
-  }
-  dependOnProjects.each {
-    implementation project(it)
-  }
-  shadow library.java.vendored_guava_26_0_jre
+  implementation project(path: ":model:fn-execution", configuration: "shadow")
+  implementation project(path: ":model:pipeline", configuration: "shadow")
```

Comment:
I would expect this to be provided

Reply:
I've moved all transitive dependencies of `:sdks:java:core` to provided.